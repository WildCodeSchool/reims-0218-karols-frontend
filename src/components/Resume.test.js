import React from "react"
import ReactDOM from "react-dom"
import Resume from "./Resume"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<Resume />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<Resume />).toJSON()
  expect(tree).toMatchSnapshot()
})
