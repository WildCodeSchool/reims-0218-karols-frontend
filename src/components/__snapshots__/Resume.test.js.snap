// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="jumbotron"
    style={
      Object {
        "backgroundColor": "#d7dbe2",
        "borderRadius": "100px",
        "fontSize": "25px",
        "textAlign": "center",
      }
    }
  >
    <h1
      className="display-12"
    >
      Récapitulatif
    </h1>
    <p
      className="shop"
    >
      Votre réservation se fera à 
    </p>
    <hr
      className="my-2"
    />
    <p
      className="préparation"
    >
      Vous avez choisi 
       
    </p>
    <p
      className="firstName"
    >
      Prénom :
    </p>
    <p
      className="gender"
    >
      Pour 
       
    </p>
    <p
      className="prestation"
    >
      Vous voulez 
       
      <br />
      Vous voulez 
       
    </p>
    <p
      className="horaire"
    >
      Vous désirez être pris en charge le 
       
    </p>
    <p
      className="lead"
    />
  </div>
</div>
`;
